File: Power Apps Project 1/Control Tower/Models/Passenger.cs
namespace Control_Tower.Models
{
    public class Passenger
    {
        public int ID { get; set; }
        public string Name { get; set; } = String.Empty;
        public string Job { get; set; } = String.Empty;
        public string Email { get; set; } = String.Empty;
        public int Age { get; set; }
        public int FlightID { get; set; }

        public ICollection<Flight> Flight { get; set; } = null!;
    }
}

File: control-tower-frontend/src/view-models/passenger.ts
// Passenger model
export interface Passenger
{
   id : number,
   name : string,
   job : string,
   email : string,
   age : number
}

// Useful type for flight data queries
export type PassengerDataParams =
{
   name : string | null,
   job : string | null,
   email : string | null,
   age : number | null,
   flightID : number | null
}